Servers/services/markdowns/projectRiskMarkdown.ts

```
@@ -8,20 +8,23 @@
-  ) : Promise<any> {
-  let rows: string;
-  const reportData = await getProjectRisksReportQuery(projectId);
-
-  if (reportData.length > 0) {
-    rows = reportData.map((risk: { risk_name: any; risk_owner: any; risk_severity: any; likelihood: any; approval_status: any; risk_level_autocalculated: any; deadline: { toLocaleDateString: () => any; }; }) => 
-      `| ${risk.risk_name} | ${risk.risk_owner} | ${risk.risk_severity} | ${risk.likelihood} | ${risk.approval_status} | ${risk.risk_level_autocalculated} | ${risk.deadline.toLocaleDateString()} |`
-    ).join('\n');
-  } else {
-    rows = `| - | - | - | - | - | - | - |`
-  }
+  ) : Promise<String> {
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)